custom_components/button_plus/button.py

```diff
@@ -107,3 +132,24 @@ def device_info(self) -> DeviceInfo:
     async def async_press(self) -> None:
         """Handle the button press."""
         _LOGGER.debug(f"async press from mqtt button: {self._btn_id}")
+
+    async def _async_long_press_action(self) -> None:
+        self._attr_click_type = "long"
+        await super()._async_press_action()
+
+    async def _async_press_action(self) -> None:
+        self._attr_click_type = "single"
+        await super()._async_press_action()
+
+    async def _async_release_action(self) -> None:
```

Comment:
Why do we have this defined and passing the body. Is the abstract function throwing errors?

Reply:
Haha no, it's just that the functionality to do something with "release" is not fully implemented yet. It's the foundational layer.